libmpv/stream_cb.h

```diff
@@ -170,6 +183,7 @@ typedef struct mpv_stream_cb_info {
     mpv_stream_cb_seek_fn seek_fn;
     mpv_stream_cb_size_fn size_fn;
     mpv_stream_cb_close_fn close_fn;
+    mpv_stream_cb_cancel_fn cancel_fn;
```

Comment:
Ideally, there should be a comment that says since which API version this field exists.